en/flight_modes/mission.md

```diff
@@ -0,0 +1,28 @@
+# MISSION Flight Mode
+
+The MISSION flight mode executes a [predefined mission/flight plan](../flying/missions.md) that has been uploaded to the flight controller. 
+
+> **Note** 
+>  * This mode requires GPS.
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Additional Detail
+
+Missions are usually created in [QGroundControl](https://docs.qgroundcontrol.com/en/PlanView/PlanView.html) and uploaded prior to launch (they may also be created in another GCS or by a developer API, and/or uploaded in flight). 
+
+If MISSION mode is activated when no mission is stored, the vehicle will enter [HOLD mode](../flight_modes/hold.md). 
+
+Missions can be paused by activating [HOLD mode](../flight_modes/hold.md). The mission will then continue from the current mission command when you reactivate the MISSION flight mode.
+
+Missions will complete and reset once the vehicle is disarmed. If the mission completes before disarming it will enter [HOLD mode](../flight_modes/hold.md) (in this case the mission is not reset).

```

Comment:
@AndreasAntener I know that if the vehicle disarms the mission will reset to the first step. Do you happen to know if there is any way to reset a mission "manually" back to start point - so for example say I finish all the steps but I'm still flying (or I've landed but not disarmed) and I want to start the mission again..